### ISSUE TITLE
Screen reader does not read that we have moved to the selection toolbar or the buttons within it

### ISSUE THREAD

**Issue by mlb76** [qlik-oss/sn-table#323]
Screen reader does not read that vi have moved to the selection toolbar or the buttons within it

**Comment by a-m-dev** [comment]
this one is a bit vague and unclear i think. 

Here is why: 
1. the selection toolbar in outside of table that we don't have any control on it (provided by nebula || provided by sense-client)
2. if we do so in any case, there is no 100% guaranty that we are exactly moving to the selection toolbar after tabbing out of the table , maybe user want's to click outside of the table while has some selections going on. we could not announce that you have moved to the selection tab while he/she may not

i think this one is a bit unclear and should be handled outside of the table not from inside of it, because simply we dont have access to the environment that table is going to be used (from inside of table)
